x/evm/artela/contract/store.go

```diff
@@ -167,6 +210,20 @@ func (k *AspectStore) StoreAspectProperty(ctx sdk.Context, aspectId common.Addre
 		ethlog.Info("StoreAspectProperty, key:", string(aspectPropertyKey), value)
 		aspectPropertyStore.Set(aspectPropertyKey, []byte(value))
 	}
+
+	// store AspectPropertyAllKey
+	keyAry := make([]string, keySet.Size())
+	for i, key := range keySet.Values() {
+		keyAry[i] = key.(string)
+	}
+	join := strings.Join(keyAry, types.AspectPropertyAllKeySplit)
+	aspectPropertyKey := types.AspectPropertyKey(
+		aspectId.Bytes(),
+		[]byte(types.AspectPropertyAllKeyPrefix),
+	)
+	aspectPropertyStore.Set(aspectPropertyKey, []byte(join))
```

Comment:
rename this store, this should not be called as aspect property store any more, it saves too many other stuffs